Jellyfin.Plugin.OpenSubtitles/OpenSubtitleDownloader.cs

```diff
@@ -30,7 +30,7 @@ public class OpenSubtitleDownloader : ISubtitleProvider
     private LoginInfo? _login;
     private DateTime? _limitReset;
     private DateTime? _lastRatelimitLog;
-    private IReadOnlyList<string>? _languages;
```

Comment:
Why is this change needed?

Reply:
`error CA1859: Change type of field '_languages' from 'System.Collections.Generic.IReadOnlyList<string>?' to 'System.Collections.Generic.List<string>?' for improved performance (https://learn.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1859)`